packages/react-google-maps-api/src/GoogleMap.tsx

```diff
@@ -141,18 +180,20 @@ export class GoogleMap extends PureComponent {
     )
   }
 
-  fitBounds = (...args) => this.state.map.fitBounds(...args)
+  fitBounds = (bounds: Bounds, padding?: number | google.maps.Padding) => this.state.map.fitBounds(bounds, padding)
 
-  panBy = (...args) => this.state.map.panBy(...args)
+  panBy = (x: number, y: number) => this.state.map.panBy(x, y)
 
-  panTo = (...args) => this.state.map.panTo(...args)
+  panTo = (latLng: LatLng) => this.state.map.panTo(latLng)
 
-  panToBounds = (...args) => this.state.map.panToBounds(...args)
+  panToBounds = (latLngBounds: Bounds, padding?: number | google.maps.Padding) => this.state.map.panToBounds(latLngBounds, padding)
 
   getBounds = () => this.state.map.getBounds()
 
   getCenter = () => this.state.map.getCenter()
 
+  // Function exists but it missing in typings
+  //@ts-ignore
```

Comment:
No need to split into two lines :)
```suggestion
  //@ts-ignore Function exists but it missing in typings
```